src/main/java/seedu/address/logic/commands/ListSellerCommand.java

```diff
@@ -0,0 +1,44 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.List;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.Messages;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.displayable.seller.Seller;
+
+/**
+ * Displays information of a specified seller.
+ */
+public class ListSellerCommand extends Command {
```

Comment:
Likewise, is there any reason why the name of the command was changed from ListSellersCommand to ListSellerCommand?

Reply:
Yes, since the commands only work on a single buyer/seller at a time.